NewForce-Cohort-9/tabloid-full-stack-debugging-dynasty
# View User Profiles

# Description

A User will be able to see a list of Users. 
[Trello Ticket #24](https://trello.com/c/dPa63gLH)

# Testing Instructions

- git add and git commit whatever you're working on on your current branch.
- git checkout main
- pull any changes to main and get all branches.
- git checkout cl-userprofile
- If you do not have Tabloid MVC database please use these [SQL Seeds](https://github.com/NewForce-Cohort-9/tabloid-full-stack-debugging-dynasty/tree/main/SQL)
- After you have done these steps run the main code in VS Community (Purple)
- Then run "npm start" for your front end code.
- Once you get into the main page login with this email "[foo@bar.com](mailto:foo@bar.com)"
- When you are logged in you will click on the "Users: at the top of the page on the navbar and you will be able to see the Users listed.



# Checklist:
